src/dialogs/more-info/ha-more-info-history.ts

```diff
@@ -114,15 +118,20 @@ export class MoreInfoHistory extends LitElement {
   }
 
   private async _getLogBookData() {
-    const yesterday = new Date(new Date().getTime() - 24 * 60 * 60 * 1000);
+    const lastDate =
+      this._lastLogbookDate ||
+      new Date(new Date().getTime() - 24 * 60 * 60 * 1000);
     const now = new Date();
+
```

Comment:
You should concat the new with the old data when we get updated data

Reply:
`this._entries = [...(this._entries || []), ...newEntries];`

Reply:
Whats the best way to concat why keeping the same logic on Entries being undefined until we make the data call